packages/main/src/services/logger/index.ts

```diff
@@ -6,10 +6,25 @@ import * as rts from 'rotating-file-stream';
 
 timber.hookConsole();
 
+const logNameGenerator = (time, index) => {
+  const pad = num => (num > 9 ? '' : '0') + num;
+  if (!time) {
+    return path.join(app.getPath('userData'), 'logs', 'nuclear-error.log');
+  }
+
+  const month = time.getFullYear() + '' + pad(time.getMonth() + 1);
+  const day = pad(time.getDate());
+  const hour = pad(time.getHours());
+  const minute = pad(time.getMinutes());
+
+  return path.join(app.getPath('userData'), 'logs', `${month}/${month}${day}-${hour}${minute}-${index}-nuclear-error.log`);
```

Comment:
Ok, that behavior looks good. Could you write a test to make sure it stays this way? You could mock `fs` and assert how the files are created.

Reply:
I have come to realize that what you said about the file names using date and time is correct. That approach does not work as intended. My apologies, I did not understand properly. 

Good thing is I think that we can just add one line argument `rotate` which seems to do the job. I have committed this change, and tested it manually. How does it sound? Also, if this change sounds good, may I ask guidance on the location to put the testing file? I was thinking to put in same folder with the index file of the logger and write a `logger.test.ts` there. Thank you for your time and patience!

Reply:
Nice, then I propose that I could try testing this manually this evening, and then if everything works as expected I could write the tests for it. I also want to replace `electron-timber` so I might do it at the same time.

Reply:
Sounds good!

Reply:
I'm struggling to test and debug this library, it's very hard to mock its `fs` calls correctly. While I have a reasonable test setup, I need to study its internal behavior, and I can't step into its `classical` function, if I set it up so that there is already a saved log file to check how it rotates, the function doesn't progress through the rest of its rotation logic. I'm aware this is testing a dependency, but I want to be sure that the way it's initialized in Nuclear's logging service is ok and works as expected, and I can't do that.